types/TypedValue.hpp

```diff
@@ -271,6 +287,30 @@ class TypedValue {
   }
 
   /**
+   * @brief Determine if the hash function getHash() is reversible for
```

Comment:
Can we expand the comments to indicate why reversible is important and how it is used?